supplier_portal/controllers/supplier_portal_website.py

```diff
@@ -0,0 +1,59 @@
+from datetime import datetime
+import base64
+
+from odoo import http
+from odoo.http import request
+
+
+class SupplierPortal(http.Controller):
+    @http.route(["/supplier/upload"], auth="user", type="http", website=True)
+    def supplier_login(self):
+        companies = request.env.user.company_ids
+
+        return request.render("supplier_portal.supplier_upload_page", {"companies": companies})
+
+    @http.route(["/supplier/submit"], auth="user", type="http", methods=["POST"], website=True)
+    def supplier_invoice_submit(self, **post):
+        try:
+            supplier = request.env.user.partner_id
+            company_id = int(post.get("company"))
+            pdf_invoice = post.get("pdf_invoice")
+            xml_invoice = post.get("xml_invoice")
+
+            if not pdf_invoice or not xml_invoice:
+                return request.redirect("/supplier/upload?error=missing_files")
+
+            # Create Vendor Bill
```

Comment:
```suggestion

            # Create Vendor Bill
```
they are set as required one so we will be able to submit only if we have those fields